layouts/_default/single.html

```diff
@@ -1,3 +1,54 @@
-{{ .Title }}
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    {{ partial "meta.html" . }}
```

Comment:
We should have a social metatags partial (see example in [reflex-template](https://github.com/opfront/reflex-template/blob/master/layouts/partials/socialMetatags.html)) so we can control how each page looks like when shared.